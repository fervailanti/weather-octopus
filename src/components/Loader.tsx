import React, { FC, ReactElement } from 'react'

const Loader: FC = (): ReactElement => (
  <div className="loader">
    <div></div>
    <div></div>
  </div>
)

export default Loader
