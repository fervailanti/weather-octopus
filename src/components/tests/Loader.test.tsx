import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Loader from '../Loader'

describe('Loader Component', () => {
  it('Should render correctly', () => {
    const loader = render(<Loader />)
    expect(loader.container).toMatchSnapshot()
  })
})
