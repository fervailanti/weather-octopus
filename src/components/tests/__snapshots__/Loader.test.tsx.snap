// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component Should render correctly 1`] = `
<div>
  <div
    class="loader"
  >
    <div />
    <div />
  </div>
</div>
`;
